File: assert_vs_match/GreenTest.java
import org.junit.*;
import static org.junit.Assert.*;

public class GreenTest {
    public void elementMustBeGreenTest() {
        Element element = new Element();
        assertEquals(Color.GREEN, element.getColor());
    }

    class Element {
        private int color;
        public Element() { color = Color.GREEN; }
        public int getColor() { return color; }
    }
}

File: assert_vs_match/green_test.rb
require "test/unit"

class GreenTest < Test::Unit::TestCase
  def test_element_must_be_green
    element = Element.new
    assert(Color::GREEN, element.color)
  end
end

class Element
  attr_accessor :color
  def initialize
    @color = Color::GREEN
  end
end

module Color
  GREEN = "#00FF00"
end
